### PR TITLE
Prepare for arrow-rs 13.0.0 release

### PR DESCRIPTION
**Pull request by alamb**
# Which issue does this PR close?

re https://github.com/apache/arrow-rs/issues/1623

# Rationale for this change
 
see subject

# What changes are included in this PR?
1. Update version
2. Changelog. See rendered version here: https://github.com/alamb/arrow-rs/blob/alamb/prepare_for_13.0.0/CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,58 @@
...
+
+**Fixed bugs:**
+
+- Parquet schema should allow scale == precision for decimal type [\#1606](https://github.com/apache/arrow-rs/issues/1606) [[parquet](https://github.com/apache/arrow-rs/labels/parquet)]
+- ListArray::from\(ArrayData\) dereferences invalid pointer when offsets are empty [\#1601](https://github.com/apache/arrow-rs/issues/1601) [[arrow](https://github.com/apache/arrow-rs/labels/arrow)]
+- Incorrect nullable flag when reading maps \( test\_read\_maps fails when `force_validate` is active\)  [\#1587](https://github.com/apache/arrow-rs/issues/1587) [[parquet](https://github.com/apache/arrow-rs/labels/parquet)]
+- Output of `ipc::reader::tests::projection_should_work` fails validation [\#1548](https://github.com/apache/arrow-rs/issues/1548) [[arrow](https://github.com/apache/arrow-rs/labels/arrow)]
+- Parquet: schema validation should allow scale == precision for decimal type [\#1607](https://github.com/apache/arrow-rs/pull/1607) [[parquet](https://github.com/apache/arrow-rs/labels/parquet)] ([sunchao](https://github.com/sunchao))
```
Comment (viirya) [inline CHANGELOG.md:47]: Isn't this the same with the first one `Parquet schema should allow scale == precision for decimal type`?

**Review by viirya — COMMENTED** [review]
Looks good except one question.

File: CHANGELOG.md (hunk 1 above)
Comment (alamb) [reply CHANGELOG.md:47]: Yes -- good call -- the script generates a line for every issue and PR marked `bug` -- I will remove the duplication. Thank you for the review @viirya

**Review by viirya — APPROVED** [review]

**Comment by alamb** [comment]
Waiting for https://github.com/apache/arrow-rs/pull/1621 and will update to get https://github.com/apache/arrow-rs/pull/1629

**Comment by alamb** [comment]
Merging this one in to create a release candidate

**Pull request merged**